packages/dnb-eufemia/src/components/pagination/Pagination.d.ts

```diff
@@ -146,6 +146,10 @@ export interface PaginationProps
    * Used during infinity mode. If `use_load_button` is set to `true`, then a button is show on the bottom. If the `startup_page` is higher than 1. Defaults to `Vis mer innhold`.
    */
   load_button_text?: string;
+  /**
+   * Used during infinity mode. Sets the icon position on the `use_load_button`. Default: `left`.
+   */
+  load_button_icon_position?: string;
```

Comment:
```suggestion
  loadButtonIconPosition?: ButtonIconPosition;
```

Needs to be imported.